pkg/schedule/schedulers/hot_region.go

```diff
@@ -219,6 +220,8 @@ var (
 	stddevThreshold = 0.1
 
 	splitBucket          = "split-hot-region"
+	splitHotReadBuckets  = "split-hot-read-region"
+	splitHotWriteBuckets = "split-hot-write-region"
```

Comment:
Does these two cases can cover the splitBucket?

Reply:
yes， use one cases can‘t distinguish the read or write load

Reply:
Then, can we remove it?

Reply:
yes ，have remove it